charts/v3.0/cray-hms-capmc/Chart.yaml

```diff
@@ -13,6 +13,6 @@ dependencies:
 maintainers:
   - name: Hardware Management
     url: https://github.com/orgs/Cray-HPE/teams/hardware-management
-appVersion: "2.4.0"
+appVersion: "2.5.0"
```

Comment:
@mjendrysik-hpe the 2.5.0 image version does not exist. Need to create the `v2.5.0` tag/release in the hms-capmc repo.

Reply:
Yeah, did that and still had to wait for the master build of CAPMC to complete. It is all good now.